### PR TITLE
Added Scout2 for security auditing

### PR DESCRIPTION
**Pull request by therandomsecurityguy**
I added Scout2 from NCC as it's actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -565,6 +565,7 @@ We cover security basics first, since configuring user accounts is something you
 -	[**AWS WAF**](https://aws.amazon.com/waf) is a web application firewall to help you protect your applications from common attack patterns.
 -	**Security auditing:**
 	-	[Security Monkey](https://github.com/Netflix/security_monkey) is an open source tool that is designed to assist with security audits.
+	-	[Scout2](https://github.com/nccgroup/Scout2) is an open source tool that uses AWS API's to assess an environment's security posture. Scout2 is stable and actively maintained.
```
Comment (jlevy) [inline README.md]: Nit: API's -> APIs

**Comment by ThanosBaskous** [comment]
LGTM - ready to merge once the "API's" -> "APIs" issue is fixed. Thanks @therandomsecurityguy!

**Comment by ThanosBaskous** [comment]
Thanks @therandomsecurityguy!

**Comment by therandomsecurityguy** [comment]
Updated API's -> APIs

Thanks all!

**Pull request merged**